internal/pkg/cli/flag.go

```diff
@@ -287,12 +298,8 @@ Uploaded asset locations are filled in the template configuration.`
 	pipelineTypeFlagDescription      = `The type of pipeline. Must be either "Workloads" or "Environments".`
 
 	// Storage.
-	storageFlagDescription               = "Name of the storage resource to create."
-	storageWorkloadFlagDescription       = "Name of the service or job to associate with storage."
-	storageLifecycleFlagDescription      = "Whether the storage should be created and deleted at the same time as a workload or as the environment"
-	storageAddIngressFromFlagDescription = `The workload that needs access to an environment-level addon.
-Must be specified with --name and --storage-type.
-Can be specified with --engine.`
+	storageFlagDescription             = "Name of the storage resource to create."
+	storageWorkloadFlagDescription     = "Name of the service/job that access the storage."
```

Comment:
```suggestion
	storageWorkloadFlagDescription     = "Name of the service/job that accesses the storage resource."
```

Reply:
<img width="557" alt="Screenshot 2023-02-17 at 3 55 40 PM" src="https://user-images.githubusercontent.com/79273084/219801281-c4b502cc-f211-4e51-8edf-fb6b8f4e3810.png">

and same here 🥲

Reply:
with resource,
<img width="567" alt="Screenshot 2023-02-17 at 3 56 09 PM" src="https://user-images.githubusercontent.com/79273084/219801337-e61538b8-6073-4329-a55f-d0a6a19b5574.png">